spec/order_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'order'
+
+describe Order do
+  subject(:order){ described_class.new(menu,text)}
+  let(:dish1){ double('menu') }
+  let(:dish2){ double('menu') }
+  let(:text){ double('text') }
+  let(:menu){ double('menu', dishes: { dish1 => 7.5, dish2 => 10.0 })}
+
+
+  describe '#initialize' do
+    it 'creates an empty order' do
+      expect(order.current.empty?).to be true
+    end
+
+    it 'initiates menu' do
+      expect(order.menu).to eq menu
+    end
+  end
+
+  describe '#show_menu' do
+    it 'shows the menu' do
+      allow(menu).to receive(:show)
+      expect(menu).to receive(:show)
+      order.show_menu
+    end
+  end
+
+  describe '#select_dishe' do
+    it 'insert a single dish with 1 in quantity into the order' do
+      order.select_dish(1,dish1)
+      expect(order.current[0]).to include 1, dish1, 7.5
+    end
+
+    it 'insert multiple dish with 1 in quantity into the order' do
+      quantity = 1
+      order.select_dish(quantity,dish1)
+      order.select_dish(quantity,dish2)
+      expect(order.current).to include [1, dish1, menu.dishes[dish1]],[quantity,dish2,menu.dishes[dish2]]
```

Comment:
Metrics/LineLength: Line is too long. [105/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)